src/Pond.tsx

```diff
@@ -32,12 +32,38 @@ type PondProps = {
   loadComponent?: JSX.Element
   /** Error callback the the pond is not able to reach actyxOS locally */
   onError?: (error: unknown) => void | JSX.Element
```

Comment:
We should use this version bump to make the onError as discussed in the last PR as required.